.github/dependabot.yml

```diff
@@ -22,14 +23,30 @@ updates:
     - kind/enhancement
     - release-note/misc
 
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 15
+    rebase-strategy: "disabled"
+    allow:
+        # cloud provider SDKs are updated too frequently, update them monthly.
+      - dependency-name: "github.com/aliyun/alibaba-cloud-sdk-go"
+      - dependency-name: "github.com/aws/*"
+      - dependency-name: "github.com/Azure/*"
+    labels:
+    - kind/enhancement
+    - release-note/misc
+
 # # # # # # # # # # # # # # # #
 #        GitHub Actions       #
 # # # # # # # # # # # # # # # #
 
   - package-ecosystem: github-actions
     directory: /
     schedule:
-      interval: daily
+      interval: weekly
```

Comment:
Same comment as for `gomod` weekly updates applies here and for the `github-actions` sections below.